src/main/daml/Daml/Finance/Interface/Settlement/Types.daml

```diff
@@ -60,13 +60,22 @@ data Approval
     --   specified instruction.
   deriving (Eq, Show)
 
--- | A unique key for Instructions.
+-- | A key for Batch.
```

Comment:
Can this live in `Daml.Finance.Settlement`? I think it's implementation specific if I am not mistaken

Reply:
See previous comment.